configure

```diff
@@ -4089,7 +4121,7 @@ case $_backend in
 		append_var LDFLAGS "-fpic"
 		_sdl=auto
 		;;
-	sdl)
+	sdl | sailfish)
```

Comment:
This looks like it doesn't get reached since sailfish is in the same case statement twice.

Reply:
Removed